### PR TITLE
Configuration of the ordered event processor via a handler. Removed diagnostics gathering. Updated docs.

### PR DESCRIPTION
**Pull request by slilichenko**

### CODE REVIEW

File: docs/pipeline-design.md [hunk 1]
```diff
@@ -66,6 +66,24 @@ ready to be processed. There are three coders used by the transform:
 * Key
   coder - [SessionContractKeyCoder in SessionContractKey class](/order-book-pipeline/src/main/java/com/google/cloud/dataflow/orderbook/SessionContractKey.java).
 
+### Create a handler to tie all the above together
+
+The OrderedEventProcessor is convigured by providing a handler which must extend
```
Comment (mescanne) [inline docs/pipeline-design.md]: configured

File: order-book-pipeline/src/main/java/com/google/cloud/dataflow/orderbook/OrderBookProcessingPipeline.java [hunk 2]
```diff
@@ -81,6 +81,12 @@ public interface Options extends GcpOptions {
     boolean isIncludeLastTrade();
 
     void setIncludeLastTrade(boolean value);
+
+    @Description("Maximum number of elements to output per each bundle. Total size of the data is not not exceed 2GB.")
```
Comment (mescanne) [inline order-book-pipeline/src/main/java/com/google/cloud/dataflow/orderbook/OrderBookProcessingPipeline.java:85]: Total size of the data is not to exceed 2GB?

Comment (slilichenko) [reply order-book-pipeline/src/main/java/com/google/cloud/dataflow/orderbook/OrderBookProcessingPipeline.java:85]: Yes, that's the hard limit of bundle output in a streaming pipeline.

**Review by mescanne — APPROVED** [review]
I suggest fixing the two typos in the comments, otherwise good to go.

**Pull request merged**